Swiftfin.xcodeproj/project.pbxproj

```diff
@@ -1490,6 +1496,16 @@
 			path = ServerDiscovery;
 			sourceTree = "<group>";
 		};
+		4EE6535F2BD3456900064B5E /* AlphaPicker */ = {
+			isa = PBXGroup;
+			children = (
+				4EE6535D2BD3456900064B5E /* AlphaPickerBar.swift */,
+				4EE6535E2BD3456900064B5E /* AlphaPickerButton.swift */,
+			);
+			name = AlphaPicker;
+			path = ../../../../../Desktop/AlphaPicker;
```

Comment:
I was just about to take a look at this but the actual work isn't checked in and lives somewhere on your machine.

At the same time, let's just keep this in `Swiftfin/Components` and probably rename to `LetterPickerBar` just to keep the name consistent with the filter. I can't remember if I said earlier "rename to alpha" so I apologize if we're back to square one.

I would like it in the following structure:
```
Swiftfin
- /Components
  - /LetterPickerBar
    - /Components
      - LetterPickerButton.swift
    - LetterPickerBar.swift
```

Reply:
Sorry about the delay on this! I have some time this weekend where I'll make the changes you were talking about. Do we want the name in code to be LetterPickerBar? I only ask because AlphaPicker seems to be the terminology used for both Web and Roku clients. I agree that the name that the end-user sees should probably be Letter Picker but I wasn't sure if we wanted everything under the hood to somewhat mirror what we were seeing on other clients? I personally have no preference so whatever you think is best!

Overall, the number of changes here is pretty small. I'll pull everything new down and copy it over. This time, actually **in** the PR... haha

Reply:
Yeah I'd like `LetterPickerBar`. I'm doing my own namings independent of what other clients do.

Reply:
This should now all exist in the PR. Sorry about that. I've changed all the naming over to LetterPicker / LetterPickerBar.

Original items still apply that I mentioned here: https://github.com/jellyfin/Swiftfin/pull/980#issuecomment-2067532843

The only new item is that last time I added the LetterPicker next to the LibraryView but I wasn't seeing that anymore in iOS? Feel free to let me know if this should be located somewhere else. It's working in testing added to PagingLibraryView but let me know if that should be somewhere else instead.